src/partitionmanager.cpp

```diff
@@ -280,9 +280,11 @@ void PartitionManagerPrivate::refresh(const PartitionList &partitions, Partition
         if (partition->status == Partition::Mounted) {
             qint64 quotaAvailable = std::numeric_limits<qint64>::max();
             struct if_dqblk quota;
+            memset(&quota, 0, sizeof(quota));
```

Comment:
just wondering: if this were c, then "struct if_dqblk quota = {};" initializer would be more correct in pedantic sense.  But c++ and what looks like c structure = idk

Reply:
Yea, could be actually better. I'll change.